code/modules/mob/mob_movement.dm

```diff
@@ -530,15 +539,25 @@
 
 	var/turf/current_turf = get_turf(src)
 	var/turf/below_turf = GET_TURF_BELOW(current_turf)
+
+	var/ventcrawling_flag = HAS_TRAIT(src, TRAIT_MOVE_VENTCRAWLING) ? ZMOVE_VENTCRAWLING : 0
```

Comment:
Put this line back where it was.

Reply:
WRT this one, I just moved it because I noticed the Up verb had the flag _there_, but I do see it's not really used if we just do relay_movement anyway!